<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng nhập để sử dụng nhà kính 4 tỷ.</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>701</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>671</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/newPrefix/glass.jpg);
border-top-left-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox#comboBox{
	background-color:rgba(0, 0, 0, 0);
	font: 14pt &quot;Segoe UI&quot;;
	color:rgba(0, 0, 0, 0);
	border: none;
	border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0.970149, y2:0.983, stop:0 rgba(170, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

</string>
     </property>
     <item>
      <property name="text">
       <string>Trần Toàn Thắng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Đinh Thanh Tùng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mai Văn Anh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nguyễn Bá Quốc Tài</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="Password">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: rgba(0, 0, 0, 0);
	font: 14pt &quot;Segoe UI&quot;;
	color: rgb(0, 120, 0);
	border: none;
	border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 229, 0, 255), stop:1 rgba(0, 121, 0, 255));
}

QLineEdit:hover{
	background-color: rgba(0, 0, 42, 80);
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Mật khẩu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EnterButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 201, 0, 255), stop:1 rgba(25, 81, 36, 255));
color: rgb(255, 255, 255);
border-radius: 20px;
</string>
     </property>
     <property name="text">
      <string>OKE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="User">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>335</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: rgba(0, 0, 0, 0);
	font: 14pt &quot;Segoe UI&quot;;
	color: rgb(0, 120, 0);
	border: none;
	border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 229, 0, 255), stop:1 rgba(0, 121, 0, 255));
}

QLineEdit:hover{
	background-color: rgba(0, 0, 42, 80);
}

</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string comment="10">Tên đăng nhập</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>671</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 80);
border-top-left-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>411</width>
       <height>671</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.97, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.502488 rgba(199, 255, 254, 242), stop:1 rgba(255, 255, 255, 255));
border-bottom-right-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	background-color: rgb(255, 10, 10);
}

QPushButton{
	border: none;
	background-color: rgba(65, 68, 65, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/Green_Icon/Icon_Green/x-square.svg</normaloff>:/Green_Icon/Icon_Green/x-square.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="BoxButton">
     <property name="geometry">
      <rect>
       <x>721</x>
       <y>160</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(191, 191, 191, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius: 2px;
}
QPushButton:hover{
	background-color: rgb(255, 255, 0);
}
QPushButton:pressed{
	background-color: rgb(255, 85, 0);
	padding-top: -2px;
	padding-left: -2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/Green_Icon/Icon_Green/chevrons-down.svg</normaloff>:/Green_Icon/Icon_Green/chevrons-down.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="forgot_password">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>290</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 120, 0);
	border: none;
}
QPushButton:hover{
	
	color: rgb(255, 85, 0);
}
</string>
     </property>
     <property name="text">
      <string>Bạn quên mật khẩu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>50</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/newPrefix/278648865_716532036438113_3320425925480882445_n-removebg-preview.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Segoe UI&quot;;
text-decoration: underline;
color: rgb(99, 255, 3);</string>
     </property>
     <property name="text">
      <string>Hãy trồng rau</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Segoe UI&quot;;
text-decoration: underline;
color: rgb(99, 255, 3);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Create_account">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 120, 0);
	border: none;
}
QPushButton:hover{
	
	color: rgb(255, 85, 0);
}
</string>
     </property>
     <property name="text">
      <string>Tạo tài khoản mới</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>400</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Bạn nhập sai mật khẩu.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eye">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>240</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:hover{
	background-color: rgba(0, 10, 10, 80);
	border-radius: 5px;
}

QPushButton{
	border: none;
	background-color: rgba(65, 68, 65, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icon.qrc">
       <normaloff>:/Green_Icon/Icon_Green/eye.svg</normaloff>:/Green_Icon/Icon_Green/eye.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>-120</x>
       <y>440</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>530</y>
       <width>291</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignTop">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#007800;&quot;&gt;Design by: &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#007800;&quot;&gt;Tứ Trụ HCMUTE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 120, 0);
background-color: rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>    1. Trần Toàn Thắng 20139015</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 120, 0);
background-color: rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>    2. Đinh Thanh Tùng 20139096</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 120, 0);
background-color: rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>    3. Mai Văn Anh 20139059</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 120, 0);
background-color: rgba(0, 0, 0, 0);</string>
        </property>
        <property name="text">
         <string>    4. Nguyễn Bá Quốc Tài 201390</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>442</x>
       <y>70</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 201, 0, 255), stop:1 rgba(25, 81, 36, 255));
background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Đăng Nhập</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_4</zorder>
    <zorder>Password</zorder>
    <zorder>EnterButton</zorder>
    <zorder>comboBox</zorder>
    <zorder>User</zorder>
    <zorder>close_button</zorder>
    <zorder>BoxButton</zorder>
    <zorder>forgot_password</zorder>
    <zorder>label_3</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>Create_account</zorder>
    <zorder>label_5</zorder>
    <zorder>eye</zorder>
    <zorder>frame_2</zorder>
    <zorder>frame_3</zorder>
    <zorder>pushButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image_main.qrc"/>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
